app/assets/javascripts/components/Columns/ColumnItem.js

```diff
@@ -1,15 +1,22 @@
 import React from "react";
```

Comment:
I think it should be on single quotes since you used single quotes on other files.